src/views/List.jsx

```diff
@@ -40,6 +41,7 @@ export function List({ data, listPath }) {
 			) : null}
 			{listPath && data.length === 0 ? (
 				<>
+					<h2>{listName}</h2>
 					<h2>
```

Comment:
Now that we've added the listName as an H2, semantically I think it would make sense to convert this empty list text to a <p> or at least an <h3>.